templates/Dataland/summary.html.twig

```diff
@@ -90,7 +90,11 @@
             <tr>
                 <td valign="top">
                     {% if dataset.researchGroup.fundingCycle %}
-                        <b>Funding Source:</b><br/>
+                        {% if custom_base_template|default == 'nas-grp-base.html.twig' %}
```

Comment:
Do we want to use this function doesTwigFileExist() from Twig/Extensions class?

Reply:
that file will exist either way, but this is what the variable gets set to, so I'm just checking for that. In the event that it does not get set, the |default == 'string' won't match, which will still be the default case.

Reply:
Yes, should we create a twig extension filter to do something similar to what we did with the Vue JS